vector/src/main/java/im/vector/app/features/settings/VectorSettingsBaseFragment.kt

```diff
@@ -148,24 +148,6 @@ abstract class VectorSettingsBaseFragment : PreferenceFragmentCompat(), Maverick
         }
     }
 
-    /**
-     * A request has been processed.
-     * Display a toast if there is a an error message
-     *
-     * @param errorMessage the error message
-     */
-    protected fun onCommonDone(errorMessage: String?) {
-        if (!isAdded) {
```

Comment:
This check is lost. Now we should probably use `launchWhenResumed`. If the operation takes time and the user leave the screen before it ends we can have crashes.

**EDIT** My bad, we are already in the `lifecycleScope`, so I guess it's fine

Reply:
yep exactly ^^^ seemed like left over code from before coroutines were introduced